src/wazuh_db/schema_upgrade_v11.sql

```diff
@@ -0,0 +1,40 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Jan 27, 2023
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+CREATE TABLE IF NOT EXISTS _sys_programs (
+    scan_id INTEGER,
+    scan_time TEXT,
+    format TEXT NOT NULL CHECK (format IN ('pacman', 'deb', 'rpm', 'win', 'pkg', 'apk')),
+    name TEXT,
+    priority TEXT,
+    section TEXT,
+    size INTEGER CHECK (size >= 0),
+    vendor TEXT,
+    install_time TEXT,
+    version TEXT,
+    architecture TEXT,
+    multiarch TEXT,
+    source TEXT,
+    description TEXT,
+    location TEXT,
+    triaged INTEGER(1),
+    cpe TEXT,
+    msu_name TEXT,
+    checksum TEXT NOT NULL CHECK (checksum <> ''),
+    item_id TEXT,
+    PRIMARY KEY (scan_id, name, version, architecture)
+);
+
+INSERT INTO _sys_programs SELECT scan_id, scan_time, format, name, priority, section, size, vendor, install_time, version, architecture, multiarch, source, description, location, triaged, cpe, msu_name, CASE WHEN checksum <> '' THEN checksum ELSE 'legacy' END AS checksum, item_id FROM sys_programs;
+DROP TABLE IF EXISTS sys_programs;
+ALTER TABLE _sys_programs RENAME TO sys_programs;
+CREATE INDEX IF NOT EXISTS programs_id ON sys_programs (scan_id);
```

Comment:
You need to create the triggers again after droping the table. See `wazuh/src/wazuh_db/schema_upgrade_v9.sql`

Reply:
I saw it and then forgot about it. Thanks sir!